README.md

```diff
@@ -1,47 +1,38 @@
-> **Notice:** *This repository hosts experimental components that are
-> currently under heavy and fast-paced development, not-ready for public
-> consumption. If you are looking for the stable registry, please head over to
-> [docker/docker-registry](https://github.com/docker/docker-registry)
-> instead.*
-
 Distribution
 ============
 
 The Docker toolset to pack, ship, store, and deliver content.
 
-The main product of this repository is the new registry implementation for
-storing and distributing docker images. It supersedes the [docker/docker-
+This repository's main product is the v2 registry implementation for
+storing and distributing Docker images. It supersedes the [docker/docker-
 registry](https://github.com/docker/docker-registry) project with a new API
 design, focused around security and performance.
 
-The _Distribution_ project has the further long term goal of providing a
-secure tool chain for distributing content. The specifications, APIs and tools
-should be as useful with docker as they are without.
-
 This repository contains the following components:
 
-- **registry (beta):** An implementation of the [Docker Registry HTTP API
-  V2](doc/spec/api.md) for use with docker 1.5+.
-- **libraries (unstable):** A rich set of libraries for interacting with
-  distribution components. Please see
-  [godoc](http://godoc.org/github.com/docker/distribution) for details. Note
-  that the libraries *are not* considered stable.
-- **dist (experimental):** An experimental tool to provide distribution
-  oriented functionality without the docker daemon.
-- **specifications**: _Distribution_ related specifications are available in
-  [doc/spec](doc/spec).
-- **documentation:** Documentation is available in [doc](doc/overview.md).
+|**Component**       |Description                                                                                                                                                                                         |
+|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **registry**       | An implementation of the [Docker Registry HTTP API,V2](doc/spec/api.md) for use with docker 1.5+.                                                                                                  |
```

Comment:
Should be "Docker Registry HTTP API V2", from the specification. This link is now broken, since the docs were moved.